scripts/stubtest/allowlist_todo.txt

```diff
@@ -244,7 +244,6 @@ django.contrib.gis.db.models.JSONField.get_transform
 django.contrib.gis.db.models.Lookup.get_prep_lhs
 django.contrib.gis.db.models.Lookup.allowed_default
 django.contrib.gis.db.models.Lookup.lookup_name
-django.contrib.gis.db.models.Lookup.output_field
```

Comment:
Most of these removals are just more cases of Django's `@cached_property` that weren't in the list that you'd already removed now that we have the workaround in mypy v1.16